code/game/objects/items/defib.dm

```diff
@@ -586,22 +586,23 @@
 				var/failed
 
 				if (H.suiciding || (HAS_TRAIT(H, TRAIT_NOCLONE)))
-					failed = "<span class='warning'>[req_defib ? "[defib]" : "[src]"] buzzes: Resuscitation failed - Recovery of patient impossible. Further attempts futile.</span>"
+					failed = "<span class='warning'>[req_defib ? "[defib]" : "[src]"] buzzes: Resuscitation failed - Genetic material makes recovery. Further attempts futile.</span>"
```

Comment:
This line doesn't make much sense to me - `Genetic material makes recovery`.
Since they're unrecoverable anyway, i think the original line made more sense.